test/integration/controllers/DeviceCommandAPI.spec.js

```diff
@@ -0,0 +1,185 @@
+const chai = require(`chai`);
+const assert = chai.assert;
+const config = require('../config');
+
+const EventEmitter = require('events');
+const events = new EventEmitter();
+
+const DeviceHive = require('../../../index');
+
+const httpDeviceHive = new DeviceHive(config.server.http);
+const wsDeviceHive = new DeviceHive(config.server.ws);
+
+const deviceId = 'e50d6085-2aba-48e9-b1c3-73c673e414be';
+
+const testDeviceCommands = [
+    {
+        deviceId,
+        command: 'command',
+        timestamp: new Date().toISOString(),
+        lastUpdated: new Date().toISOString(),
+        userId: 1,
+        networkId: 1,
+        parameters: {
+            jsonString: 'jsonString'
+        },
+        lifetime: 0,
+        status: 'status',
+        result: {
+            jsonString: 'jsonString'
+        }
+    },
+    {
+        deviceId,
+        command: 'command2',
+        timestamp: new Date().toISOString(),
+        lastUpdated: new Date().toISOString(),
+        userId: 1,
+        networkId: 1,
+        parameters: {
+            jsonString: 'jsonString'
+        },
+        lifetime: 0,
+        status: 'status',
+        result: {
+            jsonString: 'jsonString'
+        }
+    }
+];
+
+describe('DeviceCommandAPI', () => {
+
+    before(done => {
+        // Configaratuion DeviceHive
+        Promise.all([httpDeviceHive.connect(), wsDeviceHive.connect()])
+            .then(() => done());
+    });
+
+
+    it('DeviceCommandAPI.insert()', done => {
```

Comment:
Please specify in test name what should method (DeviceCommandAPI.insert()) do/return